WebHandler/__init__.py

```diff
@@ -82,55 +135,113 @@ def main_handler(req: func.HttpRequest) -> func.HttpResponse:
+                    {"status": False, "debugMessage": str(e_exception), "version": version}),
+                status_code=200
+            )
+        req_params = dict(req.params.items())
 
-    if req_params.get("method") == "advocatecasesbyname":
         @ fire_and_forget
-        def get_highcourt_cases_by_name_wrapper():
+        def get_total_no_of_cases_wrapper():
             logger.info("get_highcourt_cases_by_name_wrapper")
             try:
-                data = get_highcourt_cases_by_name(driver, req_body.get(
-                    "advocateName"), req_body.get("highCourtId"), req_body.get("benchCode"), __location__)
-                logger.info(json.dumps(data))
-                requests.post(url=req_body.get("callBackUrl"), timeout=10, json={
-                              "data": data, "request": {"body": req_body, "params": req_params}})
+                __location__ = f'{path}/{req_body.get("advocateName")}'
+                driver = create_driver(__location__)
+                case_details = get_no_of_cases(
+                    driver, req_body("advocateName"), req_body["highCourtId"], req_body["benchCode"])
+                total_cases = int(case_details["number_of_cases"][23:])
+                driver.close()
+                driver.quit()
+                if total_cases <= cases_per_iteration:
+                    start = 0
+                    stop = total_cases
+                    data = get_highcourt_cases_by_name(
+                        driver, req_body["advocateName"], __location__, start, stop)
+                    data["number_of_establishments_in_court_complex"] = case_details["number_of_establishments_in_court_complex"]
+                    data["number_of_cases"] = case_details["number_of_cases"]
+                    logger.info(json.dumps(data))
+                    requests.post(url=req_body["callBackUrl"], timeout=10, json={
+                        "data": data, "request": {"body": req_body, "params": req_params}})
+                else:
+                    n = total_cases/cases_per_iteration
+                    start = 0
+                    stop = cases_per_iteration
+                    iteration = 1
+                    while (n > 0):
+                        req_body["start"] = start
+                        req_body["iteration"] = iteration
+                        if stop > total_cases:
+                            req_body["stop"] = total_cases
+                        else:
+                            req_body["stop"] = stop
+                        logger.info({"body": req_body})
+                        try:
+                            requests.post(
+                                url="https://ecourtsapiservice-dev.azurewebsites.net/api/WebHandler?method=advocatecasesbyname", timeout=1, json=req_body)
+                        except:
+                            pass
+                        start = start + cases_per_iteration
+                        stop = stop + cases_per_iteration

```

Comment:
there should be a blank space
n = n - 1
This will be done automatically if linter works on auto format save of vs code settings.